homeassistant/components/hko/data_coordinator.py

```diff
@@ -0,0 +1,187 @@
+"""Weather data coordinator for the HKO API."""
```

Comment:
Please rename this file to `coordinator.py`.

Also please add it to `.coveragerc` since you don't have full coverage on it

Reply:
Or add tests